CustomDialogScript.rc

```diff
@@ -7,7 +7,7 @@
 //
 // Generated from the TEXTINCLUDE 2 resource.
 //
-#include "afxres.h"
+#include <winres.h>

```

Comment:
I'm curious about this. Can you explain this change further? Is there any documents I can read on this?

Reply:
I like your change. Let's go with it. I'd prefer minimal dependencies. I was just curious as to what is now different that required the change.